acorn-loose/src/statement.js

```diff
@@ -178,6 +178,12 @@ lp.parseStatement = function() {
     return this.parseClass(true)
 
   case tt._import:
+    if (this.options.ecmaVersion > 10 && this.lookAhead(1).type === tt.parenL) {
```

Comment:
Any chance we could move this into `parseExport` and do away with the `lookAhead` (by simply consuming the `import` token and looking at the next token)?

Reply:
That seems to make cases like `var dynImport = import;` work in acorn loose mode, which probably isn't desirable?

Reply:
No, that's not what I mean—inside `parseImport`, after calling `.next()` for the `import` token, you'd first look for `(`, and parse a dynamic import only if you see it (which would change the signature of `parseDynamicImport` to require a start position and expect the first token to already be consumed).